src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerAdditionService.java

```diff
@@ -127,6 +131,10 @@ private CorporateEntityBeneficialOwnerAddition getCorporateEntityBeneficialOwner
 
         corporateEntityBeneficialOwnerAddition.setCorporateName(bo.getName());
 
+        if (!bo.getTrustIds().isEmpty()) {
+                corporateEntityBeneficialOwnerAddition.setTrustIds(bo.getTrustIds());
```

Comment:
Indentation? Maybe always just set - which woul dbe set to null if getTrustIds is empty.